ExampleModels/ForwardKinematics/src/forwardKinematics.jl

```diff
@@ -0,0 +1,37 @@
+module ForwardKinematics
+
+struct Point
+    x::Float64
+    y::Float64
+end
+
+# angles is an array of angles, in order from base to end, in radians. 
+# lengths is an array of segment lengths, in order from base to end, in any (consistent) unit.
+# returns Point(x, y) of final segment. 
+# function forwardKinematics(angles::Array{Number}, lengths::Array{Number})
+function forwardKinematics(angles, lengths)
+    @assert (size(angles, 1) == size(lengths, 1)) "unequal parameter sizes"
```

Comment:
This shouldn't be an assertion, it should throw an exception.  If you're not clear on when you should assertions vs throw exceptions then look it up or ask me, but in short, @assertions should pick up invariants in your code that should only be violated if your program is incorrect